applications/vanilla/library/class.categorycollection.php

```diff
@@ -461,6 +461,42 @@ public function getTree($parentID = -1, $options = []) {
         return $tree;
     }
 
+    /**
+     * Get the id's of a category's descendants.
+     *
+     * @param int $parentIDs
+     * @param array $options
+     * @return array
+     */
+    public function getDescendantIDs($parentIDs = -1, $options = []) {
+        $ids = [];
+        $parentIDs = [$parentIDs] ?: [-1];
```

Comment:
This will never hit it's fallback condition.

Reply:
Shouldn't it be called `parentID`?

Reply:
done